kernels/tiled_add/Snakefile

```diff
@@ -0,0 +1,149 @@
+from util.snake.configs import get_snax_alu_config
+
+config = get_snax_alu_config()
+
+# Untiled example needs preprocessing:
+# FIXME: Does it really need preprocessing though?
```

Comment:
Fix? If it doesn.t, just call it .preprocfinal.mlir?

Reply:
Turns out it still needed it!